packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -294,10 +293,10 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
   /// A recognizer can be assigned to a team only when it is not participating
   /// in the arena. For example, a common time to assign a recognizer to a team
   /// is shortly after creating the recognizer.
-  GestureArenaTeam get team => _team;
-  GestureArenaTeam _team;
+  GestureArenaTeam? get team => _team;
+  GestureArenaTeam? _team;
   /// The [team] can only be set once.
-  set team(GestureArenaTeam value) {
+  set team(GestureArenaTeam? value) {
```

Comment:
Why the `?` when the next line is a non-null check?

Reply:
Oh, do getter and setter need to have the exact same type?

Reply:
>do getter and setter need to have the exact same type?

Almost. The rule is that setter can accept the return of getter. See https://github.com/dart-lang/language/issues/331